tests/integration/tier0/inhibit-if-kmods-is-not-supported/main.fmf

```diff
@@ -2,6 +2,19 @@ summary: Handle custom, tainted and force loaded kernel modules.
 description: |
     Verify, that Convert2RHEL is able to handle custom, tainted and force loaded kernel modules.
 
+adjust+:
+    # TODO disable for epel7 due to the impossibility of force loading the kmod
+    # TODO after the merge of the tier0 refactor disable just the force load kmod test
```

Comment:
Would be nice to have the names after the TODO.

Reply:
Will do, thank you @r0x0d !